scripts/gitlab-build.sh

```diff
@@ -291,11 +275,9 @@ case $BUILD_PLATFORM in
   x86_64-apple-darwin)
     STRIP_BIN="strip"
     PLATFORM="x86_64-apple-darwin"
-    EXT="pkg"
```

Comment:
Wondering where this environment variable is used.

Reply:
@5chdn looks like it was used to build up a filename [here](https://github.com/paritytech/parity/pull/8875/files#diff-3e79197c2ea5f6f50208567f975976a4L162) and afaict that's the only place.

Reply:
Yep looks like it. Perhaps we can remove `EXT` for the other platforms?